src/diffpy/structure/atom.py

```diff
@@ -47,19 +46,23 @@ class Atom:
     lattice : Lattice
         Coordinate system for the fractional coordinates `xyz`.
         When ``None`` use the absolute Cartesian system.
-
-
-    TODO: update docstrings for these properties as per
-    https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy
-
-        anisotropy   -- flag for anisotropic thermal displacements, property
-        U            -- anisotropic thermal displacement tensor, property
-        Uij          -- elements of U tensor, where i, j are from (1, 2, 3),
-                        property
-        Uisoequiv    -- isotropic thermal displacement or equivalent value,
-                        property
-        Bisoequiv    -- Debye-Waler isotropic temperature factor or equivalent
-                        value, property
+    anisotropy : Boolean
+        flag for anisotropic thermal displacements parameters. 
+        Overrides anisotropy implied from the presence of
+        ``U`` or ``Uisoequiv`` arguments.  
+        False when not specified.
+    U : numpy.ndarray
+        Anisotropic thermal displacement tensor.
+        Matrix of anisotropic displacement parameters.
+        When specified also set ``anisotropy`` = True.
+    Uij : float
+        Elements of U tensor, where i, j are from (1, 2, 3).
+    Uisoequiv: float
+        isotropic displacement parameter.
+        When specified anisotropy is set to False.
+        Only one of the ``U``, ``Uisoequiv`` arguments can be used.
+    Bisoequiv : float
+        Debye-Waler isotropic temperature factor or equivalent value.
     """
```

Comment:
Hi Pavol, sorry that I did not get a clear about "property" through the link. Could you please give me a simple example. Such as, where should I place the docstring of `anisotropy` in atom.py?